aten/src/ATen/native/BinaryOps.h

```diff
@@ -4,7 +4,13 @@
 #include <ATen/native/DispatchStub.h>
 #include <c10/core/Scalar.h>
 #include <c10/util/TypeSafeSignMath.h>
+#if defined(__CUDA_ARCH__) || defined(__HIP_ARCH__)
+#include <c10/cuda/CUDAMathCompat.h>
+#define compat_copysign c10::cuda::compat::copysign
```

Comment:
nit: instead of a macro, why not use a `using` alias?

Reply:
I found the most usages are macro, so I follow them.